packages/paste-core/components/input/src/DecrementButton.tsx

```diff
@@ -22,8 +24,17 @@ export const DecrementButton = React.forwardRef<HTMLButtonElement, DecrementButt
         size="reset"
         type="button"
         borderRadius="borderRadius20"
-        backgroundColor="colorBackground"
+        backgroundColor={props.variant === "inverse" ? "colorBackgroundInverseStrong" : "colorBackground"}
```

Comment:
nitpick: _very_ much non-blocking but when using another prop, in most cases we add it to the list of destructured props on line 17 so that we can use `variant` rather than `props.variant` (especially if we're already destructuring other specific props in the component)

Reply:
Great suggestion 🙏  I'll make a change and push it up before we merge